client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -504,8 +505,10 @@ function recordOrderInGAEnhancedEcommerce( cart, orderId, wpcomJetpackCartInfo )
  */
 function recordOrderInJetpackGA( cart, orderId, wpcomJetpackCartInfo ) {
```

Comment:
My preference would be to combine `recordOrderInJetpackGA()` and `recordOrderInWPcomGA4()` into a single function, since both functions have almost the same lines of code. The following line would be the only change needed:

```
const sendToJetpack = wpcomJetpackCartInfo.containsJetpackProducts ? true : false;

```

Reply:
This one is actually an old one: `recordOrderInJetpackGA` and also handles universal analytics. For WPcom, ecommerce has been tracked in Universal Analytics, but using the Enhanced Ecomerce function/reporting.
https://github.com/Automattic/wp-calypso/blob/892ae1abb1133596d499b1d573c21c0eb198c0ba/client/lib/analytics/ad-tracking/record-order.js#L439

So I made a new function for GA4 for WPcom, to separate the concerns.

I'll have a think about this one, and see if we can approach it differently.